src/go/k8s/cmd/configurator/main.go

```diff
@@ -281,7 +279,7 @@ func registerAdvertisedKafkaAPI(
 }
 
 func registerAdvertisedPandaproxyAPI(
-	c *configuratorConfig, cfg *config.Config, index brokerID, proxyAPIPort int,
+	c *configuratorConfig, cfg *config.Config, index int, proxyAPIPort int,
```

Comment:
this index is podOrdinal? Would. be nice to use just one name for the thing to avoid confusion

Reply:
I was thinking the same thing but was concerned about blowing up scope since this was originally wanted within the day. I think I've addressed this.